sql-cli/noxfile.py

```diff
@@ -16,19 +16,13 @@ def dev(session: nox.Session) -> None:
     development environment to ``.nox/dev``.
     """
     session.install("nox")
-    session.install("-e", "../python-sdk/.[all]")
-    session.install("-e", "../python-sdk/.[tests]")
-    session.install("-e", ".[all]")
-    session.install("-e", ".[tests]")
+    session.install("-e", ".[all,tests]")
```

Comment:
The `python-sdk` will be installed through `all`.

Reply:
The advantage of the previous way is that we'd be testing the latest version of the Python SDK, as opposed to the last released version. Perhaps we should have both, WDYT?

Reply:
Not sure if this is an advantage tbh. We should, in my opinion, treat `python-sdk` as any other package. We should only test against released stable versions.

Reply:
In my opinion testing against non-stable versions does not make sense as they could quickly change.

Reply:
So if the `python-sdk`  introduces a breaking change, the SQL CLI would only know after it is released?

Reply:
If we use proper semver in python-sdk, this is not a problem in my opinion. We could restrict the version also the same we do for other dependencies.

Reply:
I'm a -1 on this. There will be situations where we are both changing the SQL SDK and the python SDK and we will need to test the changes together.

Reply:
I agree that this may need its own PR.